cypress/e2e/inventory/cataloging/item-moved-from-one-shelf-to-another.cy.js

```diff
@@ -0,0 +1,83 @@
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import { Locations, ServicePoints } from '../../../support/fragments/settings/tenant';
+import { randomFourDigitNumber } from '../../../support/utils/stringTools';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import Users from '../../../support/fragments/users/users';
+
+describe('Inventory', () => {
+  describe('Cataloging', () => {
```

Comment:
Let's use 2 describes like
describe('inventory', () => {
  describe('Cataloging -> Maintaining the catalog', () => {

Reply:
Done